### ISSUE TITLE
Open to PRs?

### ISSUE THREAD

**Issue by cmorganBE** [someburner/pyBlufi#3]
We've got some use for this for internal testing of a blufi enabled system, this looks like the best (and maybe the only?) python blufi implementation. Would you be open to reviewing any PRs we had? I figured I'd ask before we submit any in case this isn't the case and we should instead carry our own fork.

**Comment by someburner** [comment]
yeah totally. The use / API is all experimental and fairly rough draft. Open to PRs or even recommending a maintained fork for others.

**Comment by cmorganBE** [comment]
Appreciate the work you've done here. If/when I have some suggestions for changes I'll open up some PRs for review.
